package.json

```diff
@@ -16,21 +17,31 @@
     "url": "https://github.com/freestarcapital/pubfig-adslot-react-component/issues"
   },
   "license": "ISC",
+  "peerDependencies": {
+    "react": "^15.7.0 || ^16.0",
+    "prop-types": "^15.7.2 "
+  },
   "dependencies": {
-    "@babel/runtime": "^7.12.5",
-    "@reach/router": "^1.3.4",
-    "lodash": "^4.17.20",
+    "lodash.isequal": "^4.5.0",
+    "lodash.sortby": "^4.7.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.12.1",
+    "@babel/plugin-transform-runtime": "^7.12.10",
+    "@babel/preset-env": "^7.12.11",
+    "@babel/preset-react": "^7.12.10",
+    "babel-loader": "^8.2.2",
+    "clean-webpack-plugin": "^3.0.0",
+    "np": "^7.2.0",
+    "path": "^0.12.7",
     "prop-types": "^15.7.2",
     "react": "^15.7.0",
-    "react-dom": "^15.7.0",
-    "react-scripts": "^3.4.1"
+    "webpack": "^5.15.0",
+    "webpack-cli": "^4.3.1"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "build-lib": "NODE_ENV=production && mkdir -p ./dist && rm -rf ./dist/* && npx babel src/components/FreestarAdSlot --presets=@babel/env,@babel/preset-react --plugins=@babel/plugin-proposal-class-properties,@babel/transform-runtime --out-dir dist --copy-files",
-    "publish-npm": "npm run build-lib && npm publish --access public",
```

Comment:
i see that we added `np` above too, do we not want to update this command?

Reply:
ill add docs on how to use np in line with the othe repos